Signal/src/ViewControllers/MessageDetailViewController.swift

```diff
@@ -254,11 +254,14 @@ class MessageDetailViewController: OWSViewController, MediaGalleryDataSourceDele
                     // Table view cells don't layout properly outside the
                     // context of a table view.
                     let cellView = ContactCellView()
-                    cellView.layoutMargins = UIEdgeInsets(top: 8, left: 20, bottom: 8, right: 20)
                     // We use the "short" status message to avoid being redundant with the section title.
                     cellView.accessoryMessage = shortStatusMessage
                     cellView.configure(withRecipientId: recipientId, contactsManager: self.contactsManager)
-                    groupRows.append(cellView)
+                    let wrapper = UIView()
+                    wrapper.layoutMargins = UIEdgeInsets(top: 8, left: 20, bottom: 8, right: 20)
+                    wrapper.addSubview(cellView)
+                    cellView.autoPinToSuperviewMargins()
+                    groupRows.append(wrapper)
```

Comment:
After some consideration, I favor using more wrappers.  Using the cell view's layout margins assumes that its internal layout respects margins.  Better to use composition and keep the code loosely coupled.

Reply:
I agree.